python/ray/serve/tests/test_standalone2.py

```diff
@@ -104,6 +104,28 @@ def f(*args):
     serve.shutdown()
 
 
+@pytest.mark.parametrize("detached", [True, False])
+def test_update_num_replicas_anonymous_namespace(shutdown_ray, detached):
+    """Test updating num_replicas with anonymous namespace."""
+
+    ray.init()
```

Comment:
user script's repo uses `ray.init(address="auto")`, since this test is cheap can we also parametrize this ?